src/dex-ui/services/HederaService/types.ts

```diff
@@ -28,6 +29,15 @@ interface AddLiquidityDetails {
   walletAddress: string;
   signer: HashConnectSigner;
 }
+interface TokenPairs {
```

Comment:
Minor: If this type is referring to a singular token, we can drop the 's' at the end (`TokenPair`). We may want to use the plural version for a separate type in the future.

Reply:
yes, I purposely kept it tokenPairs for now since we already have an interface with `TokenPair` and I wanted to test the functionality before changing the data in that, I will remove it in the final PR once everything is working in the app